README.md

```diff
@@ -1,135 +1,36 @@
-# Auction system
+# INTRO
 
-System architecture:
+This is a sample app of the Lagom Framework.
 
-# Item service
+## Running: Prerequisites
 
-Manages the description and auction status (created, auction, completed, cancelled) of an item.
+You will need to download and run an Elastisearch server:
 
-## Queries handled
+```
+curl -L -O https://artifacts.elastic.co/downloads/elasticsearch/elasticsearch-5.0.2.tar.gz
+tar -xvf elasticsearch-5.0.2.tar.gz
+cd elasticsearch-5.0.2/bin
+./elasticsearch
+```
+## Running
 
-* **getItem** - Gets an item by an ID.
-* **getItemsForUser** - Gets a list of items in a provided status that are owned by a given user.
+On another terminal, clone this repo and run the sample app using the command `sbt runAll`:
 
-## Events emitted
+```
+git clone git@github.com:lagom/online-auction-java.git
+cd online-auction-java
+sbt runAll
+```
 
-* **AuctionStarted** (public) - When the auction is started, in response to **startAuction**.
-* **ItemUpdated** (private) - When user editable fields on an item are updated in response to **createItem**.
-* **AuctionCancelled** (private) - When the auction is cancelled, in response to **cancelAuction**.
-* **AuctionFinished** (private) - When the auction is finished, in response to **BiddingFinished**.
+# Auction system - System architecture
 
-Event emitted publically are published via a broker topic named `item-ItemEvent`.
+The auction system is the sum of 5 micro-services and a web gateway:
 
-## Commands handled
+* **[Item Service](docs/item-service.md)**: Manages the description and auction status (created, auction, completed, cancelled) of an item.
+* **[Bidding service](docs/bidding-service.md)**: Manages bids on items.
+* **[Search service](docs/search-service.md)**: Handles all item searching.
+* **[Transaction service](docs/transaction-service.md)**: Handles the transaction of negotiating delivery info and making payment of an item that has completed an auction.
+* **user-service**: a convenience service to stub user management. Don't use any code in `user-service` as reference on how to create a secure user management micro-service.
+* **web-gateway**: a [Play](https://www.playframework.com/) application providing web UI and acting as gateway to all previously described services.
```

Comment:
I will link the code from the specialised readme pages. What I'm missing on the descriptor are the  details of the events a service will be consuming.